build.yaml

```diff
@@ -7,6 +7,7 @@ settings:
   '#3': Use "-preN" suffixes to identify pre-release versions
   '#4': Per-language overrides are possible with (eg) ruby_version tag here
   '#5': See the expand_version.py for all the quirks here
+  g_stands_for: good
```

Comment:
But I will say, "Make gRPC gRPC RPC again."

On Mon, Aug 29, 2016, 12:14 PM Eric Anderson <notifications@github.com>
wrote:

> In build.yaml
> <https://github.com/grpc/grpc/pull/7912#discussion_r76668637>:
>
> > @@ -7,6 +7,7 @@ settings:
> >    '#3': Use "-preN" suffixes to identify pre-release versions
> >    '#4': Per-language overrides are possible with (eg) ruby_version tag here
> >    '#5': See the expand_version.py for all the quirks here
> > +  g_stands_for: good
>
> To be clear: let's have 'good' here means morally good.
>
> And that avoids someone saying, "Make gRPC great again." It's always been
> great.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/grpc/grpc/pull/7912/files/8dfdb7e29912d239542c40e581660331059179a3#r76668637>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIdEkHvH0ocfl2XZoctmPoHwhSaBThdXks5qky-IgaJpZM4JvwRK>
> .
>